src/HDInsight/HDInsight/Models/Management/AzureHdInsightManagementClient.cs

```diff
@@ -121,16 +122,33 @@ public virtual OperationResource DeleteCluster(string resourceGroupName, string
             return HdInsightManagementClient.Clusters.Delete(resourceGroupName, clusterName);
         }
 
+        [Obsolete("This method has been deprecated and will stop working.")]
```

Comment:
Please do not mark internal methods as obsolete

Reply:
> Please do not mark internal methods as obsolete

Fixed. Have remove internal method.

Reply:
fixed

---
In reply to: [278675840](https://github.com/Azure/azure-powershell/pull/9069#discussion_r278675840) [](ancestors = 278675840)